src/coverlet.collector/DataCollection/CoverageManager.cs

```diff
@@ -6,18 +6,18 @@
 using Coverlet.Core.Logging;
 using Coverlet.Core.Reporters;
 
-namespace Coverlet.Collector.DataCollector
+namespace Coverlet.Collector.DataCollection
 {
     /// <summary>
     /// Manages coverlet coverage
     /// </summary>
     internal class CoverageManager
     {
-        private readonly Coverage coverage;
+        private readonly Coverage _coverage;
 
-        private ICoverageWrapper coverageWrapper;
+        private ICoverageWrapper _coverageWrapper;
 
-        public IReporter Reporter { get; }
+        public IReporter _reporter { get; }
```

Comment:
For public properties/fields PascalCase is preferred. We reserve the `_` prefixed camelCase for only private fields. Sorry we weren't clearer

Reply:
```suggestion
        public IReporter Reporter { get; }
```

Reply:
Done.

Reply:
Correct I'm adding to Toni apology  @vagisha-nidhi